deploy/examples/create-external-cluster-resources.py

```diff
@@ -559,16 +559,19 @@ def endpoint_dial(self, endpoint_str, timeout=3, cert=None):
         for prefix in protocols:
             try:
                 ep = "{}://{}".format(prefix, endpoint_str)
+                verify = None
                 # If verify is set to a path to a directory,
                 # the directory must have been processed using the c_rehash utility supplied with OpenSSL.
                 if prefix == "https" and cert and self._arg_parser.rgw_skip_tls:
+                    verify = False
                     r = requests.head(ep, timeout=timeout, verify=False)
```

Comment:
I am not python expert, but still, do we need to set `verify=False` again in the request??

Reply:
Actually it's a `key:value` argument which is passed, late on 

```
def request(self, method, url,
            params=None, data=None, headers=None, cookies=None, files=None,
            auth=None, timeout=None, allow_redirects=True, proxies=None,
            hooks=None, stream=None, verify=None, cert=None, json=None):
            ```
            
            Address by this